stripe/resource_stripe_coupon.go

```diff
@@ -188,7 +189,11 @@ func resourceStripeCouponRead(d *schema.ResourceData, m interface{}) error {
 		d.Set("metadata", coupon.Metadata)
 		d.Set("name", coupon.Name)
 		d.Set("percent_off", coupon.PercentOff)
-		d.Set("redeem_by", coupon.RedeemBy)
+
+		if coupon.RedeemBy > 0 {
+			d.Set("redeem_by", time.Unix(coupon.RedeemBy, 0).UTC().Format(time.RFC3339))
```

Comment:
Seems like with this line TF can't match this value with the value it's getting from Stripe's API.  I'd love to investigate but I won't have time right now.  I'm not merging it just yet.

Reply:
Even if imported coupon that its redeem_by is not null,  redeem_by in tfstate was set null.
Probably ignored due to type mismatch.

Reply:
Certainly this code can't care timezone.
I have 2 ideas.
1: Change type of redeem_by to TypeInt to match Stripe type.
2: Don't allow non-UTC string
However, I think it's not in this PR scope.